File: Seccion 11 - Funciones en C++/Seccion_11_Python/08_ejercicio_06.py

def maximo(dato1, dato2, dato3):
    maximo = 0

    if dato1 >= dato2 and dato1 >= dato3:
        maximo = dato1
    else:
        if dato2 >= dato1 and dato2 >= dato3:
            maximo = dato2
        else:
            maximo = dato3
    
    return maximo

dato1, dato2, dato3 = 4, 6, 8

print('El valor máximo es: {}'.format(maximo(dato1, dato2, dato3)))

File: Seccion 11 - Funciones en C++/Seccion_11_Dart/08_ejercicio_06.dart

dynamic maximo(dynamic dato1, dynamic dato2, dynamic dato3) {
  dynamic max;

  if ((dato1 >= dato2) && (dato1 >= dato3)) {
    max =dato1;
  } else {
    if ((dato2 >= dato1) && (dato2 >= dato3)) {
      max = dato2;
    } else {
      max = dato3;
    }
  }

  return max;
}

void main() {

  int dato1 = 4, dato2 = 6, dato3 = 8;

  print('El valor máximo es: ${maximo(dato1, dato2, dato3)}');
}